docs/30DaysOfPWA/advanced-capabilities/04.md

```diff
@@ -4,12 +4,148 @@
+
+![Twitter displayed in its own window on Android](_media/display-standalone.png)
+
+### Fullscreen display mode
+Fullscreen display mode takes up the entirety of the display area available and hides all of the browser UI elements. If fullscreen mode isn’t available, it will fall back to standalone. Here's an example of a game that uses `fullscreen`.
+
+![This PacMan game is a PWA and was built with the fullscreen display mode](_media/display-fullscreen.png)
+
+### Minimal-UI display mode 
+The minimal UI display mode gives your PWA a similar experience to the standalone display mode. It opens in its own window, but the application retains a minimal set of the browser UI, and the UI that is retained varies between browsers. The OneDrive PWA uses the `minimal-ui` display mode.
+
+![The OneDrive PWA utilizes the minimal-ui display mode](_media/display-minimal-ui.png) 
+
+### Browser display mode
+The browser display mode retains the browser experience with all the browser UI and your web app will not be installable. It will open in a normal browser window or tab. If Twitter were not using standalone mode, it would display in the browser like a normal tab. 
+
+![Twitter in a browser window.](_media/display-desktop.png)
+
+### Targeting different display modes with the `display-mode` media feature
+Depending on the display mode you choose for your PWA there may be elements you want to hide when the web app is open in the browser tab, especially if you’re launching in a mode like standalone or fullscreen that hides the browser UI button. You need to ensure there’s a way for people to go back if there’s no back button. Or maybe you want to modify your site styles slightly to display better when the app is in fullscreen mode. 
+
+You can do this by targeting certain display modes with the `display-mode` media feature. 
+
+```
+.app-button {
+
+    display: none;
+
+}
+
+@media (display-mode: standalone) {
+
+.app-button {
+
+    display: block;
+
+    }
+
+}
+```
+
+In this example, the button with the class .app-button would only be shown when the PWA is in standalone mode. You can go even further to combine media queries to change how things are displayed on smaller screens versus a desktop when a user in standalone mode to tailor the experience.
+
+## Customizing display options further with Window Controls Overlay 
+The previous display modes I mentioned currently only offer customization of the display below the title bar when in standalone mode. Window Controls Overlay is an experimental display option that will allow you to customize the title bar of your PWA with CSS and JavaScript. It provides access to the space next to the controls that minimize, maximize, or close the app’s window. 
+
+Because this feature is experimental and in development, the way it currently works is subject to change. You can turn this feature on in Chrome and Edge by going to `about://flags` and enabling the `Desktop PWA Window Controls Overlay` flag.
```

Comment:
```suggestion
Because this feature is experimental and in development, the way it currently works is subject to change. You can turn on this feature in Chrome and Edge by going to about://flags and [enabling the "Desktop PWA Window Controls Overlay" flag](about://flags/#enable-desktop-pwas-window-controls-overlay).
```